package.json

```diff
@@ -19,7 +19,10 @@
     "node": ">=8"
   },
   "homepage": "https://mozilla.github.io/delivery-dashboard/",
+  "homepage_stage": "https://mozilla.github.io/delivery-dashboard/stage/",
   "homepage_dev": "https://mozilla.github.io/delivery-dashboard/dev/",
+  "pollbot": "https://pollbot.services.mozilla.com/v1",
+  "pollbot_stage": "https://pollbot.stage.mozaws.net/v1",
```

Comment:
I don't think there's a good reason to list these (2 lines above) here at all. The DD stage should use Pollbot Prod. And the Pullbot URL is already mentioned inside the code.